modules/post-restore-cleanup-script.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/backing_up_and_restoring/restoring-applications.adoc
+
+:_content-type: REFERENCE
```

Comment:
If you're posting the script in line, you don't need the module.

Reply:
@kalexand-rh 
Removed